### PR TITLE
chore(rbac): Rename useryaml rbac field to authz

### PR DESCRIPTION
**Pull request by vpsx**
...in backwards compatible fashion

PXP-3071

### Improvements
Rename useryaml rbac field to authz

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
The style in this PR agrees with [`black`](https://github.com/ambv/black). :heavy_check_mark:

This formatting comment was generated automatically by a script in [uc-cdis/wool](https://github.com/uc-cdis/wool).

File: gen3users/conversion.py [hunk 1]
```diff
@@ -160,29 +160,33 @@ def convert_old_user_yaml_to_new_user_yaml(user_yaml, dest_path=None):
...
 
     # convert resources
     existing_resources = [
-        item.get("name") for item in new_user_yaml["rbac"]["resources"]
+        item.get("name") for item in new_user_yaml["authz"]["resources"]
     ]
-    old_resources = old_user_yaml.get("rbac", {}).get("resources", [])
+    if "authz" in old_user_yaml:
+        old_resources = old_user_yaml.get("authz", {}).get("resources", [])
+    else:
```
Comment (paulineribeyre) [inline gen3users/conversion.py]: You could use some kind of "authz_or_rbac" here too

**Review by paulineribeyre — COMMENTED** [review]
This works but I'm thinking it would be easier to remove the checks later if you just do `dict["authz"] = dict.get("rbac")` if `authz not in dict` once in the top level function for validate and once in the top level function for convert

**Comment by vpsx** [comment]
O yas thank you 🙇 🙇

**Review by paulineribeyre — APPROVED** [review]
lgtm

**Pull request merged**